docs/quickstart/access-control/aws-iam-eks.mdx

```diff
@@ -0,0 +1,330 @@
+
+<details>
+<summary>How to set up an AWS EKS cluster using eksctl</summary>
+
+
+Save this `yaml` as `cluster-config.yaml`:
+
+```yaml
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: otterize-iam-eks-tutorial
+  region: us-west-2
+  version: "1.27"
+
+iam:
+  withOIDC: true
+
+vpc:
+  clusterEndpoints:
+    publicAccess: true
+    privateAccess: true
+
+addons:
+  - name: vpc-cni
+    version: 1.14.0
+    attachPolicyARNs: #optional
+    - arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy
+    configurationValues: |-
+      enableNetworkPolicy: "true"
+  - name: coredns
+  - name: kube-proxy
+
+managedNodeGroups:
+  - name: small-on-demand
+    amiFamily: AmazonLinux2
+    instanceTypes: [ "t3.large" ]
+    minSize: 0
+    desiredCapacity: 2
+    maxSize: 6
+    privateNetworking: true
+    disableIMDSv1: true
+    volumeSize: 100
+    volumeType: gp3
+    volumeEncrypted: true
+    tags:
+      team: "eks"
+```
+
+Then run the following command to create your cluster. [Don't have eksctl? Install it now.](https://eksctl.io/installation/)
+
+```shell
+eksctl create cluster -f cluster-config.yaml
+```
+
+</details>
+
+Don't forget to configure your kubeconfig for your cluster. If using the example cluster above, use this command:
+```bash
+aws eks update-kubeconfig --region us-west-2 --name otterize-iam-eks-tutorial
+```
+
+#### 2. Deploy Otterize for AWS IAM
+To deploy Otterize, head over to [Otterize Cloud](https://app.otterize.com) and create a Kubernetes cluster on the [Clusters page](https://app.otterize.com/clusters), and follow the instructions. *Make sure to enable enforcement mode for this tutorial.*
+
+Once you have Otterize deployed, create an AWS IAM integration on the [Integrations page](https://app.otterize.com/integrations). Under "AWS EKS cluster name", fill in the name of your cluster on AWS EKS. In "AWS EKS cluster region", fill in the cluster's region.
+
+If you are using the cluster from the previous step, the cluster name is `otterize-iam-eks-tutorial` and the region is `us-west-2`.
+
+Once the AWS integration is configured, you'll be presented with a dialog to launch a CloudFormation template to configure the IAM roles for the Otterize operators. This enables them to manage IAM for the pods in your cluster.
+This setup is required once per-cluster, and configures AWS IRSA on your cluster, if it is not yet configured. [This template can be found on GitHub as well](https://github.com/otterize/setup/blob/main/aws-iam-operator-setup-template.yaml).
+
+After CloudFormation has configured your cluster, redeploy Otterize with the new Helm configuration provided by Otterize Cloud. Since you now have the AWS integration enabled, you need to redeploy Otterize with the credentials operator enabled, and with configuration telling the operators which AWS roles they will AssumeRole to in order to operate.
```

Comment:
Do I have to go click on the `Configure` button again?

Reply:
clarified